cmd/fetch.go

```diff
@@ -109,8 +139,8 @@ func (w *writerWorkerStruct) card() {
 			continue
 		}
 		var buffer bytes.Buffer
-		cardName := fmt.Sprintf("%v-%v%v-%v.json", card.Set, card.Side, card.Release, card.ID)
-		dirName := filepath.Join(card.Set, fmt.Sprintf("%v%v", card.Side, card.Release))
+		cardName := fmt.Sprintf("%v-%v-%v.json", card.Set, card.Release, card.ID)
+		dirName := filepath.Join("cards", config.languageCode, card.Set, card.Release)
```

Comment:
What do you think about an option to write in a specific folder ?

Reply:
Done.

Reply:
Mmmm I meant put everything you added in the option, I don't want to change my folders for my periodic task.

I think it's better if I resolved the discutions myself.